addon/templates/components/markers-styles/image/icon-editor.hbs

```diff
@@ -2,15 +2,28 @@
   <thead>
     <tr>
       <th class="six wide column">
-        {{iconCaption}}
+        
+        {{iconCaption}}        
+        
+        {{#if allowDisabling}}
+        
+        {{flexberry-checkbox 
+        value=_disabled
+        class="rightFloated"
+        }}  
+              
+        {{/if}}
+        
       </th>
-      <th class="ten wide column">
+      <th class="ten wide column" {{action 'doSomething'}}>
         {{anchorCaption}}
       </th>
     </tr>
   </thead>
-  <tbody>
+  {{#if _disabledBody}}
```

Comment:
Must be `{{#if (not _disabled)}} ... {{#if}}`, see comments above about `_disabled` and `_disabledBody` properties.